src/engine/renderer/tr_backend.cpp

```diff
@@ -3955,6 +3955,13 @@ static void RB_RenderDebugUtils()
 		static const vec3_t mins = { -8, -8, -8 };
 		static const vec3_t maxs = { 8,  8,  8 };
 
+		/* GLSL shader isn't built,
+		likely because reflection mapping is disabled. */
+		if ( gl_reflectionShader == nullptr )
```

Comment:
How about skipping only the `if ( r_showCubeProbes->integer )` block, and not the whole rest of the function

Reply:
Ah yes, I was cheated by those long blocks and I forgot they were not functions, also the next `return` cheated me. One day I'll probably turn those blocks into functions because that 1300 line function is just insane.

Reply:
Fixed.